src/theory/evaluator.cpp

```diff
@@ -259,6 +259,12 @@ EvalResult Evaluator::evalInternal(TNode n,
           break;
         }
 
+        case kind::UMINUS:
+        {
+          const Rational& x = results[currNode[0]].d_rat;
+          results[currNode] = EvalResult(-x);
+        }
+        break;
```

Comment:
Maybe move this `break` inside of the block above to be consistent with the other code.